[BUG] Language server won't start

**Describe the bug**
When launching the language server, a module not found error is thrown.

Cannot find module 'core/logger'

**To Reproduce**
Steps to reproduce the behavior:

1. Download the binary version 12.19.0 or clone the repository
2. Run: ./bp lang --offline --dim 300 --langDir ./tmp
3. See error

**Expected behavior**
The server should start properly

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment (please complete the following information):**

- OS: Linux and Windows
- Browser Chrome
- Browser Version 89.0.4389.90
- Botpress Version 12.19.0
- NodeJS 12.13.0

--- a/src/bp/lang-server/index.ts
+++ b/src/bp/lang-server/index.ts
@@ -1,5 +1,5 @@
 import chalk from 'chalk'
-import { centerText } from 'core/logger'
+import { centerText } from 'core/logger/utils'
 import _ from 'lodash'
 import path from 'path'
 

--- a/src/bp/nlu-server/index.ts
+++ b/src/bp/nlu-server/index.ts
@@ -6,7 +6,7 @@ import bytes from 'bytes'
 import chalk from 'chalk'
 import cluster from 'cluster'
 
-import { centerText } from 'core/logger'
+import { centerText } from 'core/logger/utils'
 import { copyDir } from 'core/misc/pkg-fs'
 import _ from 'lodash'
 import Engine from 'nlu-core/engine'

